.azure-pipelines/run-test-elastictest-template.yml

```diff
@@ -395,14 +377,17 @@ steps:
       inlineScript: |
         set -e
         echo "Try to cancel test plan $TEST_PLAN_ID, cancelling finished test plan has no effect."
+
+        # If TEST_PLAN_TYPE is NIGHTLY, skip the cancel step
```

Comment:
PR test is possible that pass an empty string, will it affect the logic of PR tests?

Reply:
No, empty string is considered as non-NIGHTLY.